articles/resources/relnotes.md

```diff
@@ -17,6 +17,15 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version X.XX.XXXXXXXX
+
+*Release date: November 23rd, 2020*
+
+- Improved compiler performance due to faster reference loading.
+- Updated Q# grammar for language repository. Fixed lexer bug with double literals and ranges. Updated grammar for namespaces, namespace elements, statements, and types and labels are used consistently. 
+- Updated the [`Microsoft.Quantum.Preparation` namespace](xref:Microsoft.Quantum.Preparation) to be more consistent with style guide and API design principles, and to support purified mixed states with additional data (see [proposal](https://github.com/microsoft/QuantumLibraries/issues/344) and [review notes](https://github.com/microsoft/QuantumLibraries/blob/main/Design/meetings/2020/api-design-2020-11-05.md)). https://github.com/microsoft/QuantumLibraries/pull/212, https://github.com/microsoft/QuantumLibraries/pull/322, https://github.com/microsoft/QuantumLibraries/pull/375, https://github.com/microsoft/QuantumLibraries/pull/376
+- Q# ergonomic improvements: eliminate the need for parentheses around subsequent calls. 
```

Comment:
I think "ergonomic improvements" may be a confusing term if you aren't already familiar with it, and the change can be described without it:

```suggestion
- Parentheses around repeated call expressions are now optional: `(Foo(x))(y)` may be written as `Foo(x)(y)`.
```